src/JavascriptTestTrait.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Drupal\commerce;
```

Comment:
@mglaman like core core? I thought you just meant commerce core :P Is there an existing issue or anything for core?